src/TagDependencyTrait.php

```diff
@@ -58,17 +73,27 @@ public function objectCompositeTag()
             return [];
         }
 
-        $cacheFields = (is_array($cacheFields) && !empty($cacheFields) && is_array($cacheFields[0])) ? $cacheFields : [$cacheFields];
         $tags = [];
 
         foreach ($cacheFields as $tagFields) {
             $tag = [];
+            $changed = false;
 
             foreach ($tagFields as $tagField) {
                 $tag[$tagField] = $this->$tagField;
+                $changed |= isset($oldfields[$tagField]);
```

Comment:
In this case `$changed` always equals false because `$oldfields` still is undefined